email.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const nodemailer = require('nodemailer');
+
+module.exports = (from, to, subject, text, host, port) => {
+    const mailOptions = {
+        from,
+        to,
+        subject,
+        text
+    };
+
+    const smtpConfig = {
+        host,
+        port
+    };
+
+    const transporter = nodemailer.createTransport(smtpConfig);
+
+    return new Promise((resolve, reject) => {
+        transporter.sendMail(mailOptions, (error) => {
+            if (error) {
+                reject(error);
+            }
+
+            resolve('success');
```

Comment:
`return` statement here also.

It's a preferred practice that you choose to `return` when you want to explicitly stop the execution from proceeding forward. Mainly it's for 1) ease of understanding the code (reading code is a lot harder than writing) and 2) avoids weird side-effects from occurring when someone else is making changes to this code.